xhr.bs

```diff
@@ -1457,28 +1471,26 @@ resources using <a>utf-8</a>.
 <dl class=domintro>
  <dt><code><var>client</var> . <a method for=XMLHttpRequest>overrideMimeType(<var>mime</var>)</a></code>
  <dd>
-  <p>Acts as if the `<code>Content-Type</code>` header for <a>response</a> is <var>mime</var>.
+  <p>Acts as if the `<code>Content-Type</code>` header value for <a>response</a> is <var>mime</var>.
+  (It does not actually change the header though.)
 
   <p>Throws an "{{InvalidStateError!!exception}}" {{DOMException}} if <a>state</a> is <i>loading</i>
   or <i>done</i>.
 </dl>
 
-<p>The
-<dfn method for=XMLHttpRequest><code>overrideMimeType(<var>mime</var>)</code></dfn>
-method must run these steps:
+<p>The <dfn method for=XMLHttpRequest><code>overrideMimeType(<var>mime</var>)</code></dfn> method,
+when invoked, must run these steps:
 
 <ol>
  <li><p>If <a>state</a> is <i>loading</i> or <i>done</i>, then <a>throw</a> an
  "{{InvalidStateError!!exception}}" {{DOMException}}.
 
- <li><p>Set <a>override MIME type</a> to `<code>application/octet-stream</code>`.
-
- <li><p>If <var>mime</var> is a <a>parsable MIME type</a>, then set <a>override MIME type</a> to its
- <a>MIME type portion</a>.
- <!-- XXX Ignore string to byte sequence conversion issues until some point in the future -->
+ <li><p>Set <a>override MIME type</a> to the result of <a lt="parse a MIME type">parsing</a>
+ <var>mime</var>.
 
- <li><p>If <a>override MIME type</a> has a `<code>charset</code>` parameter, then set
- <a>override charset</a> to its value.
+ <li><p>If <a>override MIME type</a> is failure, then set <a>override MIME type</a> to a new
+ <a>MIME type</a> whose <a for="MIME type">type</a> is "<code>application</code>" and
+ <a for="MIME type">subtype</a> is "<code>octet-stream</code>".
```

Comment:
You could probably also use the literal syntax here (even though it doesn't really exist).